helm-chart-sources/logstream-leader/README.md

```diff
@@ -45,6 +45,7 @@ This section covers the most likely values to override. To see the full scope of
 |---|----|-------------|
 |config.adminPassword|String|The password you want the admin user to have set.|
 |config.token|String|The auth key you want to set up for worker access. The Cribl Stream instance is configured as a distributed leader server only if this value is set. (This can, of course, also be configured via the Cribl Stream UI.) |
+|config.leaderUrl|String|When setting config.token, the env vars CRIBL_DIST_MASTER_URL and CRIBL_DIST_LEADER_URL are set, this value allows overriding the default string. This can allow you to enforce TLS leadercoms or other settings configured with this string. |
```

Comment:
The goal here is to use _tls://_ in URL or there are other reasons to override leader URL?
If so, I would suggest to introduce a new variable in values like leaderProtocol (may be not the best name) to use:
`"%s://%s@%s:4200" .Values.config.leaderProtocol .Values.config.token .Values.config.token .Values.config.bindHost`

Reply:
Since instance.yml is ignored when CRIBL_DIST_* environment vars, all settings available in instance.yml (cert locations, tls version, mTLS etc) may need to be configured here